templates/consumption/forgotten.html.twig

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div class="flex flex-col w-1/2 mx-auto pt-8">
+        <div class="-my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+            <div class="py-2 align-middle inline-block min-w-full sm:px-6 lg:px-8">
+                <div class="flex pb-4">
+                    <h2 class="flex-1 text-xl font-semibold">Consumpties</h2>
+                </div>
+                {# TODO fix below #}
```

Comment:
Moet dit nog gefixt worden, of is dat voor later?

Reply:
Volgens mij is dit klaar. TODO kan weg